llvm/utils/release/build_llvm_release.bat

```diff
@@ -163,6 +163,8 @@ set common_cmake_flags=^
   -DCLANG_ENABLE_LIBXML2=OFF ^
   -DCMAKE_C_FLAGS="%common_compiler_flags%" ^
   -DCMAKE_CXX_FLAGS="%common_compiler_flags%" ^
+  -DLLVM_ENABLE_RPMALLOC=ON ^
+  -DLLVM_ENABLE_PDB=ON ^

```

Comment:
Why ENABLE_PDB?

Reply:
Removed, I had it for debugging. Will propose that separately.